pyproject.toml

```
@@ -118,5 +118,15 @@ checks = ["all", #report on all checks, except the following
 exclude = ['__init__' ] # don't report on objects that match any of these regex
 
 [tool.mypy]
+disable_error_code = ['type-arg', 'import-untyped', 'import-not-found', 'no-untyped-call']
+strict = true
+explicit_package_bases = true
 follow_imports = 'skip'
-exclude = ["tests", "docs"]
+exclude = ["tests", "docs" , "__init__.py"]
+mypy_path = ["imap_processing/" , "tools/"]
```

I think this should be under packages, but it was also a little unclear to me what `mypy_path` is supposed to do. It sounds more like it is trying to locate the configuration and mypy specific things rather than the packages to test.
```suggestion
packages = ["imap_processing" , "tools"]
```